website/docs/tutorials/06-deploying/02-migrations/invoke.md

```diff
@@ -5,8 +5,8 @@ def invoke(
     contract_address: int,
     function_name: str,
     inputs: list[int] | dict[str, Any] | None = None,
+    config: SignedCheatcodeConfig,
     *,
```

Comment:
I think `config` should be below the `*`. It's a keyword only argument.

Reply:
[reorder args so config is keyword-only](https://github.com/software-mansion/protostar/pull/979/commits/4867cf55bafc6a338417888275d52879bf529e2a)